src/proof/dot/dot_printer.h

```diff
@@ -73,9 +75,17 @@ class DotPrinter
    * @return The string with the special characters escaped.
    */
   static std::string sanitizeString(const std::string& s);
+
+  /** Traverse proof node and populate map of subproofs to their quantity
```

Comment:
This is counting the # subproofs underneath each subproof of pn? 

Can you state this?  I had initially thought this was mapping each subproofs to the number of times it occurs.

Reply:
Updated the doc, PTAL